LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2015–17 Hack Cambridge
```

Comment:
There is some guidance on it [here](https://www.gnu.org/licenses/gpl-howto.html) relating to GPL but should be relevant to this case too. The dash means that each year is copyrightable, i.e., significant changes were made each year between those two years. Yeah you're right 2015 is good, I'd forgotten this was branched from the original site! 
So all good. As you were!